doc/install_mne_c.rst

```diff
@@ -2,290 +2,162 @@
 
-Unpack the tar archive:
+To install from the compressed tar archive, change directory to the desired
+install location, and unpack the software using ``tar``:
 
 .. code-block:: console
 
-    $ tar zxvf <software package>
+    $ cd /path/to/install/location
+    $ tar zxvf <path_to_archive_file>
 
-The name of the software directory under <*dir*> will
-be the same as the package file without the .gz extension.
-
-Installing from a Mac OSX disk image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-- Double click on the disk image file.
-  A window opens with the installer package ( <*name*> .pkg)
-  inside.
-
-- Double click the the package file. The installer starts.
-
-- Follow the instructions in the installer.
-
-.. note::
-
-    The software will be installed to /Applications/ <*name*> by default.
-    If you want another location, select Choose Folder... on the Select a
-    Destination screen in the installer.
+To install from the macOS disk image, double-click the downloaded .dmg file. In
+the window that opens, double-click the installer package file (.pkg) to launch
+the installer, and follow its instructions.
 
 .. _user_environment:
 
-Setting up MNE-C environment
-############################
-
-Your system-dependent path to the MNE Software will be referred to by
-the environment variable MNE_ROOT.
-
-Your system-dependent path to MATLAB Software will be referred to by
-the environment variable MATLAB_ROOT.
+Configuration
+^^^^^^^^^^^^^
 
-You can download a free runtime distribution of Matlab with the libraries
-necessary to run Matlab code from,
-https://www.mathworks.com/products/compiler/matlab-runtime.html
+MNE-C requires two environment variables to be defined manually:
 
-If you do not have a Matlab distribution, then leave MATLAB_ROOT undefined.
-Please note however that if Matlab is not available, the utilities
-mne_convert_mne_data, mne_epochs2mat, mne_raw2mat, and mne_simu will not work.
+- ``MNE_ROOT`` should give the path to the folder where MNE-C is installed
+- ``MATLAB_ROOT`` should give the path to your MATLAB binary (e.g.,
+  ``/opt/MATLAB/R2018b`` or similar).  If you do not have MATLAB or the MATLAB
+  runtime, leave ``MATLAB_ROOT`` undefined.
 
-To setup the MNE environment, a script must be sourced into your current shell.
-The result will be that additional environment variables like MNE_LIB_PATH
-will be set in your current shell in order to run the MNE Software.
-
-A note about source-ing scripts with "source" and "." (dot). The hash bang at
-the top of a script tells exec what shell to use by default to interpret the
-script. But the hash bang line is ignored when you source a script via either
-the builtin "source" or "." directive.  So there needs to be a setup script
-compatible with your current shell, i.e., if you use csh you cannot source a
-bash script.  (BTW, the "." (dot) directive will search for the script in
-$PATH if there is no slash in the script argument while "source" does not
-search $PATH).
```

Comment:
this paragraph is general software information / advice, not specific to MNE.  If a user follows the instructions, they don't actually need to know this.